src/widgets/PageInfo.js

```diff
@@ -129,14 +137,14 @@ export const PageInfo = props => {
             }
           >
             <View>
-              {columnData.map((...args) =>
-                renderTitleComponent(isEditingDisabled, columnIndex, ...args)
-              )}
+              {columnData
+                .filter(data => data.shouldShow !== false)
```

Comment:
If am not a fan of default value = true for optional props/keys. In my opinion it' better to use falsy, i.e. shouldHide, then you don't need to !== false. Keeping optionals as falsy would avoid ppl making mistakes when changing code in the future

Reply:
This was to avoid side effects in the rest of the app, not having to go and set `shouldShow` on everywhere that uses `PageInfo`. I think best to keep it like this, make a new issue to fix it?

Reply:
done 20821301679666ffe2b38fb80267fac54d73c1b9